tests/sample/data_usage/associatetoflow.c

```diff
@@ -0,0 +1,27 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include "ebpf_helpers.h"
+#include "ebpf_nethooks.h"
+
+#pragma clang section data = "maps"
+ebpf_map_definition_t app_map = {
+    .size = sizeof(ebpf_map_definition_t),
+    .type = EBPF_MAP_TYPE_HASH,
+    .key_size = sizeof(five_tuple_t),
+    .value_size = sizeof(uint64_t),
+    .max_entries = 500};
+
+
+#pragma clang section text = "flow"
```

Comment:
You need same prefix to share maps

Reply:
Same prefix is needed if both maps are in same ELF file, in this case, we are not combining the programs because we are doing a 2-map approach. The eBPF programs do not need to access each other's maps.

Reply:
Got it. U r right